conf/messages

```diff
@@ -107,8 +107,8 @@ label.sign_in_details=Sign-in details
 label.your_sign_in_details=Your sign-in details
 label.sign_in_details_content=Government Gateway user ID and password you use to sign-in
 label.your_gg_details=your Government Gateway details
-label.do_you_live_in_the_uk=Do you live in the UK?
-label.do_you_live_in_the_uk.change=do you live in the UK?
+label.do_you_live_in_the_uk=Is your main address in the UK?
```

Comment:
Shall we change the label of msg as well
label.do_you_live_in_the_uk to label.is_your_main_address_in_the_uk

Reply:
done